package.json

```diff
@@ -161,42 +161,42 @@
     "karma-webpack": "3.0.5",
     "less": "^2.7.2",
     "less-loader": "4.1.0",
-    "loader-utils": "^1.1.0",
+    "loader-utils": "1.2.3",
     "md5": "^2.2.1",
     "minimist": "^1.2.0",
     "mountebank": "^1.15.0",
-    "ngtemplate-loader": "^1.3.1",
+    "ngtemplate-loader": "2.0.1",
     "node-libs-browser": "^2.0.0",
     "physical-cpu-count": "^2.0.0",
     "postcss-colorfix": "^0.0.4",
-    "postcss-loader": "^2.1.2",
+    "postcss-loader": "3.0.0",
     "prettier": "^1.14.3",
     "pretty-quick": "^1.4.1",
     "react-addons-test-utils": "15.6.2",
     "react-test-renderer": "16.3.2",
     "request-promise-native": "^1.0.5",
     "rimraf": "^2.5.4",
-    "style-loader": "^0.20.3",
-    "thread-loader": "^1.1.5",
-    "ts-loader": "4.3.0",
+    "style-loader": "0.23.1",
+    "thread-loader": "2.1.2",
+    "ts-loader": "5.3.3",
     "ts-node": "^7.0.1",
     "tslint": "^5.11.0",
     "tslint-config-prettier": "^1.14.0",
     "tslint-eslint-rules": "^5.3.1",
     "tslint-loader": "^3.6.0",
     "tslint-react": "^3.6.0",
     "typescript": "^3.2.4",
-    "uglifyjs-webpack-plugin": "^1.2.4",
-    "url-loader": "^1.0.1",
+    "uglifyjs-webpack-plugin": "2.1.1",
```

Comment:
Do we still need this if we are using terser?

Reply:
nope, good catch